### ISSUE TITLE
Search bar glitches out on mobile

### ISSUE THREAD

**Issue by YourLocalStore** [RoVSRG/robeats-cs#9]
While playing the beta version of the rewrite on mobile, the characters start glitching in the search bar (but that's only when you start trying to delete any characters you typed). Might be an issue on my end, but that's what someone said as well on the RCS group wall.

https://user-images.githubusercontent.com/84473197/119216114-affb9900-ba9f-11eb-89d1-fd8adf32cd44.mp4

(this was also relevant in the alpha build as well)

**Comment by thenerdie** [comment]
Oh my, thats a weird bug. Will investigate this soon. Does this happen all of the time, or does it happen occasionally?

**Comment by YourLocalStore** [comment]
> Oh my, thats a weird bug. Will investigate this soon. Does this happen all of the time, or does it happen occasionally?

This definitely happens all the time, I rejoined a new server around two times and tried it again and happened to be the same result. It's pretty weird. Though one way I found out to go around this was to exit the keyboard, enter it again, delete the characters and repeat until it's all gone. But it may be tedious for other users I'd say.

**Comment by thenerdie** [comment]
> > Oh my, thats a weird bug. Will investigate this soon. Does this happen all of the time, or does it happen occasionally?
> 
> This definitely happens all the time, I rejoined a new server around two times and tried it again and happened to be the same result. It's pretty weird. Though one way I found out to go around this was to exit the keyboard, enter it again, delete the characters and repeat until it's all gone. But it may be tedious for other users I'd say.

Yeah that would be rather tedious. Does this only happen on mobile?

**Comment by hamburg21035** [comment]
It only happens on mobile, but I‘ve seen this in a lot of other games. It is rather a issue in the Roblox Mobile App or something is wrong with the code.

**Comment by astralkingdummy** [comment]
A fix for this will be provided next week.

**Comment by ryandoesalot** [comment]
This still has not been fixed

**Comment by astralkingdummy** [comment]
Ah crap, i'll try to find a fix as soon as possible
